lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
+  <%%= f.error_notification %>
+
+  <div class="form-inputs">
+  <%- attributes.each do |attribute| -%>
+    <%%= f.<%= attribute.reference? ? :association : :input %> :<%= attribute.name %> %>
+  <%- end -%>
+  </div>
+
+  <div class="form-actions">
+    <%%= f.button :submit %>
+  </div>
+<%% end %>
```

Comment:
This file was generated by Simple Form...I've never customized [scaffold templates](http://guides.rubyonrails.org/generators.html#customizing-your-workflow-by-changing-generators-templates) before, and am somewhat indifferent about whether we want to keep it or not. Thoughts?